kubejs/client_scripts/ponder/base.js

```diff
@@ -1,22 +1,50 @@
 // priority: 10
 
-onEvent('ponder.tag.registry', (event) => {
-    event.create(
-      'enigmatica:industrial_foregoing',
-      'industrialforegoing:latex_processing_unit',
-      'Industrial Foregoing',
-      'Learn more about the Industrial Foregoing mod and its intriguing mechanics!'
-    );
-    event.create(
-      'enigmatica:basics',
+const categories = [
+  {
+    id: 'enigmatica:industrial_foregoing',
+    cat_item_id:'industrialforegoing:latex_processing_unit',
```

Comment:
please be explicit with variable names, `category_item_id` would be better imo :)